README.md

```diff
@@ -25,7 +25,7 @@ Properties consumed (in `bootstrap.properties`):
 
 - `embedded.container.redis.enabled` (default is `true`)
 - `embedded.container.redis.startup-timeout` (default is `30`)
-- `embedded.container.redis.docker-image` (default is `redis:6.2.6-alpine`)
+- `embedded.container.redis.docker-image` (default is `redis:6.2.11-alpine`)
```

Comment:
Muss auch in der additional-spring-configuration-metadata.json angepasst werden. Gilt für alle PRs.

Reply:
Auch in den anderen PRs angepasst.